### ISSUE TITLE
RAM is not freed when opening new images within a project

### ISSUE THREAD

**Issue by GXcells** [qupath/qupath#1646]
I am using the portable version QuPath-v0.5.0-Windows, on a 64bit Windows machine.

In a project with several images (each between 1.5 and 2GB), when I open one image then open a new image, the RAM used from 1st image stays and is not "freed".

When looking at the Memory monitor of QUpath, if I press "clear tile cache", it will decrease the "Used memory", but the "total memory" stays the same.

So basically, even when clearing tile cache, the RAM is taken up is growing little by little at a point where I have to close Qupath and restart.

I don't think that this is an expected behavior. 
Also if just going back and forth between the same 2 images, loading is faster if tiled are created (normal) but it will  increases the number of cached tiles and increases total RAM (but at some point not anymore, which is even more strange). 

Reclaim memory also does not completely free the RAM.

**Comment by petebankhead** [comment]
QuPath is a Java application, and the memory behavior is complicated and implementation-dependent: https://www.baeldung.com/gc-release-memory

The memory monitor uses the `Runtime` class, and makes calls to `System.gc()` for garbage collection.
The [javadocs](https://docs.oracle.com/en/java/javase/22/docs/api/java.base/java/lang/Runtime.html#freeMemory()) contain statements like `"Calling the gc method may result in increasing the value returned by freeMemory`"... where that 'may' is interesting.

In some cases (such as the tile cache), QuPath retains a (soft or weak) reference to objects for 'as long as it can' to help performance, but will release them before any out-of-memory error is produced.

In short, this doesn't sound like a bug to me. Unless there can be shown to be a memory leak within QuPath itself, there are too many other possible reasons beyond our control why memory use reported by the JVM might increase.
